### PR TITLE
Move analysis failures into skipped section of text output

### PR DESCRIPTION
**Pull request by ievans**
PR checklist:

- [x] Documentation is up-to-date
- [x] Changelog is up-to-date
- [x] Change has no security implications (otherwise, ping security team)

### CODE REVIEW

File: semgrep/semgrep/target_manager.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def converted_pipe_targets(targets: Sequence[str]) -> Iterator[Sequence[str]]:
 
 
 @define
-class IgnoreLog:
+class IgnoreOrFailedLog:
```
Comment (underyx) [inline semgrep/semgrep/target_manager.py]:
```suggestion
class Log:
```
or
```suggestion
class TargetingLog:
```
?

Comment (ievans) [reply semgrep/semgrep/target_manager.py]: What's your thinking here @underyx? I'd prefer more verbose, but I guess `TargetingLog` is just more generic? Maybe `FileTargetingLog`?

File: semgrep/semgrep/cli.py [hunk 2]
```diff
@@ -14,7 +14,6 @@ def cli() -> None:
     """
     To get started quickly, run `semgrep scan --config auto`
     """
-    pass
```
Comment (ievans) [inline semgrep/semgrep/cli.py:17]: pre-commit did this

**Review by adamboros — APPROVED** [review]
Lgtm, small nit that it says

> Scan skipped: 1 **files**

but singular/plural is handled here: 

> Ran 3 rules on 1 **file**: 0 findings.

**Comment by ievans** [comment]
Agreed @adamboros unfortunately that's an inherent issue, can save for a future PR

**Pull request merged**